mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/repositories/TiltaksgjennomforingRepository.kt

```diff
@@ -851,4 +857,35 @@ class TiltaksgjennomforingRepository(private val db: Database) {
 
         return tx.run(queryOf(query, mapOf("id" to id, "status" to status.name)).asUpdate)
     }
+
+    fun lukkApentForInnsokForTiltakMedStartdatoForDato(dagensDato: LocalDate): Int {
+        @Language("PostgreSQL")
+        val query = """
+            update tiltaksgjennomforing
+            set apent_for_innsok = false
+            where oppstart = 'FELLES' and start_dato = ? and apent_for_innsok = true
+            returning id
+        """.trimIndent()
+
+        val gjennomforinger = queryOf(query, dagensDato).map {
+            it.uuid("id")
+        }.asList.let { db.run(it) }
+
+        gjennomforinger.forEach {
+            endringshistorikkService.logEndring(
+                documentClass = DocumentClass.TILTAKSGJENNOMFORING,
+                operation = "Satt 'Åpent for innsøk = false' via automatisk jobb",
+                userId = "System",
+                documentId = it,
+                timestamp = LocalDateTime.now(),
+            ) {
+                Json.parseToJsonElement(
+                    """
+                    {"tiltaksgjennomforingId": "$it" }
+                    """.trimIndent(),
+                )
+            }
+        }
```

Comment:
Usikker på denne. Tror det kan være kjekt å se i endringsloggen hvorfor åpent for innsøk automatisk har endret seg, men om den bør ligge i repo-klassen her vet jeg ikke helt.